Apps/Admin/Client/Components/BroadcastDialog.razor.cs

```diff
@@ -73,9 +73,9 @@ public partial class BroadcastDialog : FluxorComponent
 
     private bool IsNewBroadcast => this.Broadcast.Id == Guid.Empty;
 
-    private bool HasAddError => this.BroadcastsState.Value.Add.Error != null && this.BroadcastsState.Value.Add.Error.Message.Length > 0;
+    private bool HasAddError => this.BroadcastsState.Value.Add.Error is { Message.Length: > 0 };
```

Comment:
can we discuss pattern usage in code? I personally find these unreadable.

Reply:
I've added this to the list to discuss